### PR TITLE
Rm command

### PR DESCRIPTION
**Pull request by aureliencarle**
+ stable remove for files
+ add convert with compact retrun in PasswordIO
+ add in Shell : add_password and save.
+ for now the save is make when exit

### CODE REVIEW

**Comment by aureliencarle** [comment]
It's now tag as draft feel free to change and merge ... maybe better to discuss first

File: src/session.py [hunk 1]
```diff
@@ -91,6 +90,12 @@ def get_key(self, password: str) -> bytes:
     def generate_path(self) -> str:
         return os.path.join(self.passpath, self.name)
 
+    def add_password(self, password: PasswordData):
+        if password.label not in self.content.keys():
+            self.content.update({password.label: password})
```
Comment (GregoireUhlrich) [inline src/session.py]: Maybe replace by 
```
self.content[password.label] = password
```
Equivalent but more clear

File: src/commands/rm.py [hunk 2]
```diff
@@ -8,14 +8,21 @@ class Rm:
 
     @staticmethod
     def do(shell: Shell, line: str):
+        parameter, _ = shell.get_arguments(line)
```
Comment (GregoireUhlrich) [inline src/commands/rm.py]: Replace `_` by `options`, the underscore was meant for cases in which the options are not used (avoid an unused variable)

**Comment by GregoireUhlrich** [comment]
Still the `destroy()` method to implement, but this is on the right tracks :)

File: src/session.py [hunk 3]
```diff
@@ -66,12 +66,12 @@ def print_content(self, key: str, is_secure: bool) -> None:
...
-        pass
-        #pathfile = self.passpath+'/'+self.name
-        #if os.path.exists(pathfile):
-        #   os.remove(pathfile)
-        #else:
-        #   Io.print('File does not exist')
+        pathfile = self.passpath + '/' + name
+        if os.path.exists(pathfile):
+            os.remove(pathfile)
+            self.files = os.listdir(CRYPTPATH)
```
Comment (GregoireUhlrich) [inline src/session.py:72]: Could replace by `self.files.remove(pathfile)`
This should be equivalent and not require an os call to list the remaining files.

**Pull request merged**